commands_buy.go

```diff
@@ -58,5 +59,49 @@ func (b buyCmd) ToAuditEntry() string {
 }
 
 func (b buyCmd) Execute() {
-	consoleLog.Warning("Not implemented: BUY")
+	consoleLog.Debug("Starting Buy")
+	if _, accountExists := accountStore[b.userID]; !accountExists {
+		consoleLog.Errorf("No account for %s exists", b.userID)
+		return
```

Comment:
Throughout the entire PR. `Execute()` should only finish by reaching the end or through an `abort` function.

This will become a lot more useful once we start returning values. If we need to send back a response we can use `abortTx` and friends to generate a standard error response + message, rather than trying to determine how `Execute()` finished. If `Execute()` only reaches the end via the happy path then we can build a response message around that assumption.